wdio.conf.js

```diff
@@ -49,7 +49,7 @@ exports.config = {
   // and 30 processes will get spawned. The property handles how many capabilities
   // from the same test should run tests.
   //
-  maxInstances: 10,
+  maxInstances: 1,
```

Comment:
is this to execute just one instance? if so, please add this new feature in your PR's description. So the code reviewer will be aware of this change